docs/rag.md

```diff
@@ -124,10 +124,6 @@ Embedding model support is provided through the following plugins:
 The following examples show how you could ingest a collection of PDF documents
 into a vector database and retrieve them for use in a flow.
```

Comment:
Consider adding a blurb about what the sample(s) do (read in restaurant menus and answer questions about them).

Reply:
Done. PTAL.